src/Components/categories.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Categories = () => (
+  <>
+    <h2>Hello Categories</h2>
+    <button type="submit"> Check status </button>
+    <li>The new Horizon By david Norson</li>
+    <button type="submit">remove</button>
+    <li>The lands of Arcanum By rida Bensalem</li>
+    <button type="submit">remove</button>
+  </>
```

Comment:
According to the requirement 🙍‍♀️, your category should display a button "Check status" only.
Kindly implement it.